Tools/symbolicate-ppc.py

```diff
@@ -0,0 +1,95 @@
+#!/bin/python3
```

Comment:
Should probably be /usr/bin/python3

Some distros (debian) are still clinging to the old ways.